README.md

```diff
@@ -1,50 +1,128 @@
+   ```
+
+The following environments are used to launch the Red Hat AppStudio installation in E2E mode and the tests execution(tokens are also used for running the tests):
+
+| Variable | Required | Explanation | Default Value |
+|---|---|---|---|
+| `GITHUB_TOKEN` | yes | A github token used to create AppStudio applications in github  | ''  |
+| `QUAY_TOKEN` | yes | A quay token to push components images to quay.io. Note the quay token must be your dockerconfigjson encoded in base64 format, e.g. `ewogI3dJhdXRocyI6I...` | '' |
+| `GITHUB_E2E_ORGANIZATION` | no | GitHub Organization where to create/push Red Hat AppStudio Applications  | `redhat-appstudio-qe`  |
+| `QUAY_E2E_ORGANIZATION` | no | Quay organization where to push components containers | `redhat-appstudio-qe` |
+| `E2E_APPLICATIONS_NAMESPACE` | no | Name of the namespace used for running HAS E2E tests | `appstudio-e2e-test` |
+| `PRIVATE_DEVFILE_SAMPLE` | no | The name of the private git repository used in HAS E2E tests. Your GITHUB_TOKEN should be able to read from it. | `https://github.com/redhat-appstudio-qe/private-quarkus-devfile-sample` |
+
+3. Install dependencies:
 
 ``` bash
 # Install dependencies
 $ go mod tidy
+# or go mod tidy -compat=1.17
 # Copy the dependencies to vendor folder
 $ go mod vendor
-# Create `e2e-appstudio` binary in bin folder. Please add the binary to the path or just execute `./bin/e2e-appstudio`
-$ make build
 ```
 
-## Install App Studio in e2e mode
+4. Install Red Hat AppStudio in e2e mode with install script in scripts folder. The e2e framework will use by default the `redhat-appstudio-qe` GitHub organization(you can change the GitHub organization by environment variable `GITHUB_E2E_ORGANIZATION`).
+
+   ```bash
+      make local/cluster/prepare
+   ```
+   or
+   ```bash
+       ./scripts/install-appstudio-e2e-mode.sh install
+    ```
+
+More information about how to deploy Red Hat AppStudio
+are in the [infra-deployments](https://github.com/redhat-appstudio/infra-deployments) repository.
+
+## Building and running the e2e tests
+You can use scripts to build and run the tests:
+   ```bash
+      make local/test/e2e
+   ```
+Or build and run the tests without scripts:
+1. Install dependencies and build the tests:
+
+   ``` bash
+   # Install dependencies
+   $ go mod tidy
+   # Copy the dependencies to vendor folder
+   $ go mod vendor
+   # Create `e2e-appstudio` binary in bin folder. Please add the binary to the path or just execute `./bin/e2e-appstudio`
+   $ make build
+   ```
+
+2. Run the e2e tests:
+The `e2e-appstudio` command is the root command that executes all test functionality. To obtain all available flags for the binary please use `--help` flags. All ginkgo flags and go tests are available in `e2e-appstudio` binary.
 
-To install Red Hat App Studio in e2e mode you can found the instructions in [docs folder](https://github.com/redhat-appstudio/e2e-tests/tree/main/docs)
+   ```bash
+    `./bin/e2e-appstudio`
```

Comment:
Just wondering whether we should push more for using Ginkgo CLI instead of building and running the test binary.
When we get to the point when we can run tests in parallel, it will be possible to run it only via ginkgo cli, wdyt?

Reply:
Right now I would keep both options in docs, since we are using the e2e binary in our scripts for OpenShift CI, but we can start propagating Ginkgo CLI since we are trying to run the tests in parallel.